src/web/routes/application/steps/address/request-body.js

```diff
@@ -1,9 +1,11 @@
-const { ADDRESS_KEYS } = require('./constants')
-
-const getAddressFieldFromClaim = (claim) => (address, key) => ({ ...address, [key]: claim[key] })
-
 const requestBody = (session) => ({
-  address: ADDRESS_KEYS.reduce(getAddressFieldFromClaim(session.claim), {})
+  address: {
```

Comment:
Previous code was getting all of the address fields as entered, which meant the non sanitised postcode was being added to the request body.

Reply:
Not sure what the answer should be, but should we be using the ADDRESS_KEYS from constants here?

Reply:
Updated to use the address constants.